src/pages/schedule/schedule.ts

```diff
@@ -85,7 +83,10 @@ export class SchedulePage {
   goToSessionDetail(sessionData) {
     // go to the session detail page
     // and pass in the session data
-    this.navCtrl.push(SessionDetailPage, sessionData);
+    this.navCtrl.push(SessionDetailPage, {
+      data: sessionData,
+      session: sessionData.name
```

Comment:
This was part of the changes needed for adding the deeplinker. Id be in favor of keeping this change for now so that when deeplinker is ready and we add it again I can just drop in the config.